versioned_sidebars/version-0.2.0-sidebars.json

```diff
@@ -1,6 +1,7 @@
 {
   "version2": [
     "overview",
+    "glossary",
```

Comment:
@sayantani11  - can you add glossary at the end of the sidebar, please? 

Glossary is usually added at the end of a book (similar to Index).